test/integration.sh

```diff
@@ -101,8 +102,7 @@ done
     --grpc-address=127.0.0.1:10911 \
     --http-address=127.0.0.1:9091 \
     --store=127.0.0.1:10901 \
-    --log.level=error \
-    --web.external-prefix=.
```

Comment:
I'm not entirely sure what should be the effect of setting this flag to `.`, in the latest version it seemed to break the requests to upstream, leading to `404`:
```
level=debug name=observatorium ts=2021-07-20T10:12:56.859796347Z caller=proxy.go:61 request=mgera.remote.csb/eHcoZcWYFZ-000013 msg="request to upstream" url=http://127.0.0.1:9091/api/v1/query
level=debug name=observatorium ts=2021-07-20T10:12:56.860200601Z caller=instrumentation.go:36 request=mgera.remote.csb/eHcoZcWYFZ-000013 proto=HTTP/2.0 method=POST status=404 content=application/x-www-form-urlencoded path=/api/metrics/v1/test-oidc/api/v1/query duration=851.928µs bytes=19
caller=level.go:63 ts=2021-07-20T10:12:56.860387071Z name=up level=error caller=main.go:142 component=reader msg="failed to query" err="query response parse failed: invalid character 'p' after top-level value"
```
Removing it made the requests working again.

Reply:
Yep, this was a workaround for a bug that was fixed quite a while ago in Thanos.